src/tr2/game/objects/creatures/dragon.c

```diff
@@ -48,9 +51,6 @@ typedef enum {
     DRAGON_ANIM_SWIPE_LEFT  = 9,
     DRAGON_ANIM_SWIPE_RIGHT = 10,
     DRAGON_ANIM_DEATH       = 11,
-    DRAGON_ANIM_DIE         = 21,
-    DRAGON_ANIM_DEAD        = 22,
-    DRAGON_ANIM_RESURRECT   = 23,
     // clang-format on
 } DRAGON_ANIM;
```

Comment:
Yes I just moved these out because these are animation numbers. It seems like even in TR1, the enums are all called `X_ANIM` even though they are technically animation states. And the `#defines` include the few animation numbers that might be hard coded. We could probably use a large rename in the future to `X_ANIM_STATE` probably.

Reply:
Let's either refrain from this change or make a DRAGON_STATE and DRAGON_ANIM enums.

Reply:
Ok I'll just kick the can down the line since this is going to require a rename of every object.